tests/lib/rules/no-unguarded-globals.js

```diff
@@ -504,6 +504,72 @@ ruleTester.run('no-unguarded-globals', rule, {
       errors: [{
         message: DOCUMENT_MESSAGE
       }]
+    },
+    {
+      code: `
+        import environment from 'ember-stdlib/utils/environment';
+
+        export default Ember.Component.extend({
+          shortCircuit() {
+            if (environment.isBrowser() && window.localStorage) {
+              return window.localStorage;
+            }
+          }
+        });`,
+      parserOptions: {
```

Comment:
You'll likely want to rebase and clean up these `parserOptions` usages now that the other PR has landed that removes multiple usages of them.

Reply:
Done